analysis/trajectory.Rmd

```diff
@@ -10,8 +10,77 @@ bibliography: ../ref.bib
 source("workflows/knitr_options.R")
 ```
 
-*WIP*
+In this chapter, we will walk through a minimal analysis to perform lineage reconstruction and psuedotime inference for single-cells along a continuum. 
+We will demonstrate this with the [`slinghot`](https://bioconductor.org/packages/slingshot) Bioconductor package. 
+The goal of slingshot is to use clusters of cells to uncover global structure and convert this structure into smooth lineages represented by one-dimensional variables, called "pseudotime".
+The package offers tools to learn the cluster relationships in an unsupervised or semi-supervised manner and constructing smooth curves representing each lineage, along with visualization methods for each step.
+Slingshot was designed to model developmental trajectories in scRNA-seq data and serve as a component in an analysis pipeline after dimensionality reduction and clustering. 
 
+We will use the 10X PBMC data in the Bioconductor data package `TENxPBMCData` that we have been using in previous chapters. 
+
+```{r, results='asis', echo=FALSE}
+extractCached("workflows/tenx-batch-pbmc/4k", "clustering", "pbmc4k")
+```
+
+```{r}
+pbmc4k
+```
+
+If you recall, we created our cluster labels and have three reduced dimensions in our `SingleCellExperiment` object. 
+
+```{r}
+head(pbmc4k$cluster)
+```
+
+```{r}
+reducedDimNames(pbmc4k)
+```
+
+At this point, we have everything we need to run slingshot. 
+This is a two-step process composed of identifying the global lineage structure with a cluster-based minimum spanning tree (MST) and fitting simultaneous principal curves to describe each lineage.
```

Comment:
More theoretical details. In particular, I would like to know what the advantages of this method are compared to other approaches. I would also like details on the most important tunable parameters. For example, have a look at the internal debates of pros/cons of various approaches in the clustering chapter.

Reply:
added